ui/css/treeherder.css

```diff
@@ -522,20 +522,19 @@ th-watched-repo {
     white-space: nowrap;
 }
 
-.result-set .job-list-pad-left {
-    padding-left: 8;
```

Comment:
This was unused anyway due to the syntax error, so I just removed it. We default to 15px padding on the left side of the job list.